tests/nightly/test_np_large_array.py

```diff
@@ -905,17 +905,27 @@ def batch_check(x, modes, params):
     assert type(out[0]).__name__ == 'ndarray'
 
 @use_np
-@pytest.mark.skip(reason='backward errors out')
 def test_ctc_loss():
-    A = np.ones((2, INT_OVERFLOW, 4))
+    def test_ctc_loss_size_check(A, label):
+        assertRaises(MXNetError, npx.ctc_loss, A, label)
```

Comment:
this should be ValueError

Reply:
Right! I did not know changing the error message actually controls the error type. I have changed it to ValueError.